app/src/main/java/it/ministerodellasalute/immuni/logic/exposure/repositories/ExposureReportingRepository.kt

```diff
@@ -74,4 +75,11 @@ class ExposureReportingRepository(
             storage[lastProcessedChunkKey] = value
         }
     }
+
+    val lastSuccessfulCheckDate: Date?
```

Comment:
@ioane-bendingspoons shouldn't be better to expose as LiveData or StateFlow here?

Reply:
I would use LiveData in ViewModels and UI related classes only (since they need lifecycle), StateFlow seems more appropriate.